scalalib/src/mill/scalalib/Dependency.scala

```diff
@@ -24,9 +24,17 @@ object Dependency extends ExternalModule {
     }
 
   /** Show possible dependency updates. */
-  def showUpdates(ev: Evaluator, allowPreRelease: Boolean = false): Command[Unit] = T.command {
-    DependencyUpdatesImpl.showAllUpdates(updates(ev, allowPreRelease)())
+  def showUpdates(
+      ev: Evaluator,
+      allowPreRelease: Boolean = false,
+      format: Format = Format.PerModule
+  ): Command[Unit] = T.command {
+    DependencyUpdatesImpl.showAllUpdates(updates(ev, allowPreRelease)(), format)
```

Comment:
By adding this command before the other (now deprecated) overload, Mill resolves it correctly from CLI. I'm not sure, if that is guaranteed or arbitrary. Maybe, we should detect overloads and apply some heuristics (`@deprecated`) or explicit annotation to select the right one or fail.